resources/assets/coffee/react/beatmaps/search-filter.coffee

```diff
@@ -62,9 +56,16 @@ window.FilterSelector = React.createClass
   render: ->
     selectors = []
     $.each @props.options, ((i, e) ->
-      selectors.push a href:'#', className: ('active' if @selected(e['id'])), value: e['id'], onClick: @select.bind(@, e['id']), e['name']
+      selectors.push a href:'#', className: ('active' if @selected(e['id'])), value: e['id'], key: i, onClick: @select.bind(@, e['id']), e['name']
     ).bind(this)
 
     div id: @props.id, className: 'selector', 'data-name': @props.name,
       span className:'header', @props.title
       selectors
+
+SearchFilter.propTypes =
```

Comment:
Using `@propTypes` (with `@` sign) seems to be working for me without warnings. Either you tried it with ES6 (instead of coffee) which indeed doesn't support this or react fixed it in 0.14.